client/src/components/App.js

```diff
@@ -37,6 +38,7 @@ const App = (props) => {
         <Route exact path="/users/new" component={RegistrationForm} />
         <Route exact path="/user-sessions/new" component={SignInForm} />
         <Route exact path="/home" component={HomePage} />
+        <Route exact path="/profile" component={UserProfilePage} />
```

Comment:
`AuthenticatedRoute` should be used here. Otherwise, the page will break/ not display anything if a user not signed in tries to navigate to this route